### PR TITLE
Implement Fluid, and FluidHolder

### PR DESCRIPTION
**Pull request by ProfElements**
Implements Fluid as an SlimefuItem, and FluidHolder for things like storage tanks

### CODE REVIEW

**Comment by ProfElements** [comment]
I didnt modifiy existing items like `LiquidMetal.java` because that would cause a giant rewrite and I dont want to do that to you

File: src/main/java/io/github/seggan/liquid/FluidHolder.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        Validate.notNull(l, "Location was null!");
+        Validate.isTrue(fluidAmount > 0, "The fluidAmount to remove must be greater than zero!");
+
+        try {
+            int fluidCapacity = getFluidCapacity();
+
+            if (fluidCapacity > 0) {
+                int currentFluidAmount = getFluidAmount(l);
+
+                // Check if there is even energy stored
```
Comment (Seggan) [inline src/main/java/io/github/seggan/liquid/FluidHolder.java]: energy???

File: src/main/java/io/github/seggan/liquid/FluidHolder.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+                }
+            }
+        } catch (Exception | LinkageError x) {
+            Bukkit.getLogger().log(Level.SEVERE, x, () -> "Exception while trying to set the fluid-stored for \""
+                    + /*getId()*/ "block" + "\" at " + new BlockPosition(l));
+        }
+    }
+
+}
```
Comment (Seggan) [inline src/main/java/io/github/seggan/liquid/FluidHolder.java]: Please add newlines to end of files

**Review by Seggan — CHANGES_REQUESTED** [review]
Looks good, just a few styling changes (and honestly, my system does need a rewrite)

**Review by Seggan — APPROVED** [review]
Looks good

**Comment by Seggan** [comment]
CI skipped because no new content added.

**Pull request merged**